__tests__/app.test.js

```diff
@@ -0,0 +1,64 @@
+const {topicData, articleData, userData, commentData} = require("../db/data/test-data/index")
+const request = require ("supertest")
+const app = require("../app")
+const connection = require("../db/connection")
+const seed = require("../db/seeds/seed")
+
+
+beforeEach(()=>{
+return seed({topicData, articleData, userData, commentData})
```

Comment:
Hiya, just to make things a bit clearer - we've bundled up the data in the index files so you can just go ahead and require it in as a single variable, and pass that to seed.
So instead of `const {topicData, articleData, userData, commentData} = require("../db/data/test-data/index")`, you could do `const data = require("../db/data/test-data/index")`, and instead of `return seed({topicData, articleData, userData, commentData})` you could then do `return seed(data)`

Just figured I would clear this up in case it was confusing :)